### PR TITLE
Feature/us133 digital register page printable

### PR DESCRIPTION
**Pull request by camillebaldock**
## Instructions for review

To be reviewed commit by commit. 
### Issues fixed
- removes old compressed copy/pasted CSS and introduced SASS compilation
- redirects to search page if root url is visited while user is logged in
- redirects to login page if root url is visited while user is not logged in
- forms use HTML to indicate required fields and required lengths
- validation on form matches GOV.UK design pattern
- search term are no longer transformed to CAPS by default
- Text such as `result(s)`, `Owner(s)` or `Lender(s)` are no longer pluralised by indicating the `(s)` but rather an `s` is added if the value is plural
- Breadcrumbs are made generic and can be specified from server.py
- Need to duplicate values of google_analytics_api code, and `./static` path across the app is removed
- Removes duplication of beta banner across the app
## Screenshots of new frontend

title-summary-no-map
title-summary-map
search
search-results-1
search-results
login
login-error
## Screenshots of printable summary page

printable

### CODE REVIEW

**Comment by Demotive** [comment]
First 3 commits :+1: - nice :)

**Comment by Demotive** [comment]
Haven't got this running locally, but from a front end perspective reading through the code: LGTM

:+1:

**Comment by camillebaldock** [comment]
:warning: Jenkins is down so tests aren't running :(

**Comment by camillebaldock** [comment]
goes with -> https://github.com/LandRegistry/digital-register-feeder/pull/150  :heart:

File: service/templates/display_login.html [hunk 1]
```diff
@@ -1,47 +1,72 @@
...
+              {% endfor %}
+            </ul>
+          </div>
+        {% endif %}
+        {% if unauthorised_title %}
+          <div class="error-summary" role="group" aria-labelledby="error-summary-heading-example-1" tabindex="-1">
+            <h1 class="heading-medium error-summary-heading" id="error-summary-heading-example-1">
+              {{ unauthorised_title }}
+            </h1>
+            <p>{{unauthorised_description}}</p>
```
Comment (Pheonnexx) [inline service/templates/display_login.html]: no spaces between the {{}} and the variable

File: service/templates/error.html [hunk 2]
```diff
@@ -2,26 +2,15 @@
...
-                </div>
-            </div>
+{% block body %}
+<div class="outer-block">
+    <div class="inner-block">
+        <h1 class="heading-xlarge">
+            {{error}}
+        </h1>
+        <div class="text">
+            <p>{{description}}</p><br/>
```
Comment (Pheonnexx) [inline service/templates/error.html]: no spaces between the {{}} and the variable.

**Comment by Pheonnexx** [comment]
Checked, only very minor comments.

File: service/server.py [hunk 3]
```diff
@@ -86,34 +87,20 @@ def load_user(user_id):
...
 def signin_page():
-    return render_template(
-        'display_login.html', asset_path='/static/',
-        google_api_key=GOOGLE_ANALYTICS_API_KEY,
-        form=SigninForm(csrf_enabled=_is_csrf_enabled()),
-        username=current_user.get_id()
-    )
+    user_id = current_user.get_id()
+    if user_id:
+        return redirect(url_for('find_titles_page'))
```
Comment (lgonczar) [inline service/server.py:99]: :+1:

File: service/server.py [hunk 4]
```diff
@@ -301,7 +281,6 @@ def get_property_address_index_polygon(geometry_data):
 
 
 def _get_address_search_response(search_term, page_number):
-    search_term = search_term.upper()
```
Comment (lgonczar) [inline service/server.py:304]: I hated the statefulness that this line represented. But doesn't removing it break anything? Does postcode search still work?

**Comment by lgonczar** [comment]
Looks good. Left one question.

**Comment by camillebaldock** [comment]
There is a defect on the search for title/postcode linked to remove the 'upper': will add an acceptance test and fix...please do not merge until defect fixed :(

**Comment by camillebaldock** [comment]
@russwillis fixed the link, @lgonczar fixed the search issue, will fix this as part of another PR :) realised my code only worked because I had made a code change on the API (_facepalm_)

**Pull request merged**